### ISSUE TITLE
flickering titlebar

### ISSUE THREAD

**Issue by briandconnolly** [donadigo/appeditor#103]
For the last couple releases on Fedora, AppEditor's titlebar is transparent, flickers and also shows multiple titlebars.

**Comment by ottotivadar** [comment]
Can confirm, on Fedora 33 Workstation.

**Comment by donadigo** [comment]
Are you using the Flatpak of AppEditor? If yes, then I unfortunately can't do nothing about it.

**Comment by Devorlon** [comment]
Similar problem. I'm using the appeditor-git package from the AUR. Gnome, Wayland.

**Comment by ottotivadar** [comment]
> Are you using the Flatpak of AppEditor? If yes, then I unfortunately can't do nothing about it.

No, I'm using the one from the Fedora repo: https://fedora.pkgs.org/33/fedora-x86_64/appeditor-1.1.1-2.fc33.x86_64.rpm.html

Just tested on another Fedora 33 (Gnome, Wayland too), and had similar problem.

**Comment by hapla** [comment]
I have the same problem on Fedora 33, using appeditor-1.1.1-2 rpm from Fedora repo.
Here is a screenshot from Gnome+Wayland (it looks different on my screen than on the screenshot):
![appeditor-gnome-wayland](https://user-images.githubusercontent.com/13129935/103817034-a03e6380-506e-11eb-9be3-ccfbf63fca8c.png)
Gnome+X.org looks similar (on screen).

And this is Fedora's "Gnome classic":
![appeditor-gnome-classic](https://user-images.githubusercontent.com/13129935/103817237-f14e5780-506e-11eb-974a-b4dcd1e15da1.png)

**Comment by Tuxman2** [comment]
I'm on Manjaro and I have the same problem as Devorlon and others people here. I'm on last Gnome version with Wayland.

**Comment by hapla** [comment]
Commenting out one line in appeditor/data/application.css seems to fix the problem:

@@ -26,7 +26,7 @@
 
 .titlebar {
     border: none;
-    background-image: none;
+    /* background-image: none; */
     background-color: @bg_color;
     box-shadow:

**Comment by ottotivadar** [comment]
> Commenting out one line in appeditor/data/application.css seems to fix the problem:
> 
> ```
> @@ -26,7 +26,7 @@
>  
>  .titlebar {
>      border: none;
> -    background-image: none;
> +    /* background-image: none; */
>      background-color: @bg_color;
>      box-shadow:
> ```

Can confirm, just did a fresh build on Fedora 33, and this patch solved the problem.

**Comment by musicinmybrain** [comment]
Hi, I’m the current maintainer of the `appeditor` package in Fedora. It’s helpful if you can also report any issues like this [downstream in Fedora](https://docs.fedoraproject.org/en-US/quick-docs/howto-file-a-bug/) as well so I can track them. I won’t always take time to debug nontrivial issues, but in some cases I may be able to help.

I will go ahead and apply @hapla’s patch in Fedora until upstream has a chance to revisit the issue. Expect an update to reach F33/F4 stable in a week.

**Comment by musicinmybrain** [comment]
@donadigo, do you plan to make this change upstream?

If you have an idea for a better fix, I’m happy to try it out in Fedora and see if it also fixes the issue.

**Comment by Enderteck** [comment]
> Commenting out one line in appeditor/data/application.css seems to fix the problem:
> 
> ```
> @@ -26,7 +26,7 @@
>  
>  .titlebar {
>      border: none;
> -    background-image: none;
> +    /* background-image: none; */
>      background-color: @bg_color;
>      box-shadow:
> ```

Sorry I'm kinda new to linux and especially to where our apps data are stored. Where can I find this CSS file ? And for the Flatpak version ? Thanks for any answers.

**Comment by musicinmybrain** [comment]
> I will go ahead and apply [@hapla](https://github.com/hapla)’s patch in Fedora until upstream has a chance to revisit the issue. Expect an update to reach F33/F34 stable in a week.

As noted in the discussion for https://github.com/donadigo/appeditor/pull/139, this approach doesn’t seem to work anymore.
